src/tests/test_experiments.py

```diff
@@ -232,6 +233,24 @@ def test_simple_experiment(experiment_class, matrix_storage_class):
         ))
         assert len(linked_matrices) == len(matrices)
 
+class TestConfigVersion(TestCase):
```

Comment:
This feels like something I would test with a mock. The definition of what constitutes a validation error is bleeding into here. 
```
experiment.validate = MagicMock()
experiment.run()
experiment.validate.assert_called_once()
```

Also I think the test class is misnamed

Reply:
oh, fancy mocking -- will add that in. And, yeah, copy-paste error on the test class!